pkgs/servers/openvscode-server/default.nix

```diff
@@ -150,13 +151,9 @@ in stdenv.mkDerivation rec {
   '';
 
   installPhase = ''
-    mkdir -p $out/libexec
-
-    cp -R -T ../vscode-reh-web-${vsBuildTarget} "$out/libexec"
-
-    ln -s ${nodejs}/bin/node $out/libexec
-
-    makeWrapper "$out/libexec/server.sh" "$out/bin/openvscode-server"
+    mkdir -p $out
+    cp -R -T ../vscode-reh-web-${vsBuildTarget} $out
+    ln -s ${nodejs}/bin/node $out
```

Comment:
Is there a reason to change the output folders here? Not necessarily opposed to it, but just curious, and I'd like to keep the derivation similar to `code-server` since it's doing something quite similar.

Reply:
This is to match the folder structure that the upstream project ships in their releases. Hopefully it shouldn't be a breaking change as it still retains `bin/openvscode-server` as the entrypoint.

Reply:
Cool, that sounds good 👍